dt_model/simulation/evaluation.py

```diff
@@ -0,0 +1,43 @@
+from dt_model.model.instantiated_model import InstantiatedModel
```

Comment:
Note to self, we probably need to tag before this release and then push out another release making it clear in the release note that there is a breaking change where one needs to use `simulation` instead of `ensemble`.